ansible/init.sh

```diff
@@ -62,6 +62,15 @@ main() {
         execute_playbook $env $passwd_file
     done
 
+    # Grant the pipeline service account permissions to create projects
```

Comment:
This step requires a pipelines operator to be installed on a cluster. Can you please mention it somewhere in a doc? Or maybe even better install the pipeline operator in an ansible role.

Reply:
Most of the steps (eg. installing tasks and pipelines) already required the pipeline operator. Can I create an additional ticket to do that?

Reply:
https://issues.redhat.com/browse/ISV-1115